drivers/dma/dma_intel_adsp_hda.c

```diff
@@ -465,7 +466,7 @@ void intel_adsp_hda_dma_isr(void)
 
 	if (clear_l1_exit) {
 #if CONFIG_SOC_SERIES_INTEL_ACE
```

Comment:
Since the ifdefs make these no-ops in case of not ACE then the ifdef wrapping the call can be removed

Reply:
Fixed